app/views/users/profile.html.erb

```diff
@@ -233,12 +233,13 @@
       </div>
     </div>
   <% end %>
-
+ <%if !Rails.env.production?%>
   <% if @profile_user && current_user && @profile_user == current_user %>
     <%= form_tag "/users/test_digest_email", method: :post do %>
       <%= submit_tag "Test Digest Email" %>
     <% end %>
   <% end %>
+  <% end %>
```

Comment:
- and also format the indentation for the last end statement so that it aligns with the if statement?